Source/Core/DiscIO/Filesystem.h

```diff
@@ -100,6 +100,8 @@ class FileInfo
   // to get a file and its path, building the path while traversing is faster.
   virtual std::string GetPath() const = 0;
 
+  virtual std::unique_ptr<FileInfo> CopyFileInfo() const = 0;
```

Comment:
This doesn't seem to actually be used by anything.

Reply:
It was used in the first attempt I made at VolumeFileBlobReader, which currently just returns an error saying that its copy is unimplemented. You advised me on Slack that it was unnecessary because it's not used for running games, and I just forgot that it was the only thing I needed to copy a FileInfo object for.